src/torrent_handle.cpp

```diff
@@ -513,6 +518,12 @@ namespace libtorrent
 		return r;
 	}
 
+	bool torrent_handle::keep_redundant_connections() const
```

Comment:
you added this function to a section of deprecated functions. so this won't be included when building with ``TORRENT_NO_DEPRECATE``. In fact, I think this state should be reflected in the ``torrent_status::flags`` instead, where these other states were moved.